Tools/autotest/arducopter.py

```diff
@@ -4000,13 +4000,43 @@ def fly_gyro_fft(self):
             raise ex
 
     def test_onboard_compass_calibration(self, timeout=300):
+        ofs_vals = [5, -13, 8]
```

Comment:
we need bigger offsets than that to really know it works, and we need to have different offsets on each simulated compass